virttest/utils_nbd.py

```diff
@@ -85,6 +85,8 @@ def setup_certs(self):
 
         # Get server hostname.
         hostname = process.run('hostname', ignore_status=False, shell=True, verbose=True).stdout_text.strip()
+        ip_addr_cmd = "ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*).*/\2/p' |head -n 1"
```

Comment:
As a suggestion why not start to use iproute2 utils instead of old net-tools?

Reply:
updated